quiz.rb

```diff
@@ -1,20 +1,64 @@
 # Question 1
 # Create a class called Dog. It should have, as attributes, name, breed, and age.
 # Make all of those attributes read-only.
+class Dog
+  def initialize (name, breed, age)
+    @name = name
+    @breed = breed
+    @age = age
+  end
+
+  attr_reader :name
```

Comment:
Two small points. 
1) You can actually write this in a single line by separating the symbols with commas, e.g.
    `attr_reader :name, :breed, :age`
2) It's convention to put `attr_reader